README.adoc

```diff
@@ -166,24 +151,21 @@ Navigate to the `start` directory.
 [role=command]
 include::{common-includes}/devmode-start.adoc[]
 
-Begin by building your response JSON, which is composed of the `_links` array and the name of
-the host machine.
+Begin by building your response JSON, which is composed of the name of the host machine and its list of HATEOAS links.
 
 === Linking to inventory contents
 
 As mentioned before, your starting point is an existing simple inventory REST service. 
 
-Take a look at the `request handlers` in the [hotspot file=0]`src/main/java/io/openliberty/guides/microprofile/InventoryResource.java` file.
+Take a look at the request handlers in the [hotspot file=0]`InventoryResource.java` file.
 
 InventoryResource.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
 include::finish/src/main/java/io/openliberty/guides/microprofile/InventoryResource.java[]
 ----
 
-Since the `.../inventory/hosts/`
-URL will no longer respond with a JSON representation of the contents of your inventory, you can discard the `listContents`
-method and integrate it into the [hotspot=PropertiesForHost file=0]`getPropertiesForHost` method.
+Since the `.../inventory/hosts/` URL will no longer respond with a JSON representation of the contents of your inventory, you can discard the `listContents` method and integrate it into the [hotspot=PropertiesForHost file=0]`getPropertiesForHost` method.
```

Comment:
Update

>Since the `.../inventory/hosts/` URL will no longer respond with a JSON representation of the contents of your inventory, you can discard the `listContents` method and integrate it into the `getPropertiesForHost` method.

To

>The `.../inventory/hosts/` URL will no longer respond with a JSON representation of your inventory contents, so you can discard the `listContents` method and integrate it into the `getPropertiesForHost` method.